src/webgpu/api/validation/image_copy/buffer_related.spec.ts

```diff
@@ -104,8 +104,15 @@ g.test('bytes_per_row_alignment')
       .beginSubcases()
       .combine('bytesPerRow', [undefined, 0, 1, 255, 256, 257, 512])
       .combine('copyHeight', [0, 1, 2, 4, 8])
-      // copyHeight must be aligned with blockHeight
-      .filter(({ format, copyHeight }) => copyHeight % kTextureFormatInfo[format].blockHeight === 0)
+      // copyHeight has some constraints:
+      //   - it must be aligned with blockHeight.
```

Comment:
In many tests we solve this by having the test parameters provide e.g. `copyHeightInBlocks`. Could that be done here? This works now but would break down if the texel block height were 6 for example. (which happens with ASTC)

Reply:
Done.